homeassistant/components/obihai/strings.json

```diff
@@ -14,6 +17,9 @@
           "host": "[%key:common::config_flow::data::host%]",
           "password": "[%key:common::config_flow::data::password%]",
           "username": "[%key:common::config_flow::data::username%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Obihai device."
```

Comment:
Shouldn't we link to the other string to limit translation work?

Reply:
agreed. Thank you for the fix. I didn't know how. :-). Is there a way to apply reuse so that it works for all integrations?

Reply:
We link within the same file or to topics of the core platform, but only if they are within the same context (state->state) (exception->exception). For general descriptions we have a common `strings.json` which resides at the root `homeassistant/strings.json`.